data_structures/is_even.py

```diff
@@ -0,0 +1,38 @@
+# Find if a number is even or odd,
+# without using the mod, div or multiplication operators.
+
+
+def is_even(n, option="AND"):
```

Comment:
Use Enum to decide on options not strings.

Reply:
Enum used instead of strings